tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -363,10 +366,89 @@ def check_succeed(self, validate_metrics=True):
 
         self._metrics = metrics
 
-        if validate_metrics:
+        if validate_metrics and raise_exceptions:
             OMBSampleConfigurations.validate_metrics(self._metrics,
                                                      self.validator)
 
+        if validate_metrics and not raise_exceptions:
+            is_valid, results = OMBSampleConfigurations.validate_metrics(
+                self._metrics, self.validator, raise_exceptions=False)
+            #if not raise_exceptions:
+            return is_valid, results
+
+    def detect_spikes_by_percentile(self,
+                                    results,
+                                    significant_factor=1.5,
+                                    max_spike_width=1,
+                                    expected_max_latencies=None):
+        def detect_spikes_in_series(latency_series, expected_max):
```

Comment:
would be nice to see a big high level comment here about what is going on (why we are doing this, can link to an issue) and how we are doing the detection, etc, to set the scene for the reader

Reply:
Good idea. Added.